pdctl/command/namespace_command.go

```diff
@@ -22,19 +22,20 @@ import (
 )
 
 const (
-	namespacePrefix    = "pd/api/v1/namespaces"
-	namespaceAddPrefix = "pd/api/v1/namespaces/add"
+	namespacePrefix      = "pd/api/v1/namespaces"
+	namespaceTablePrefix = "pd/api/v1/namespaces/table"
 )
 
 // NewNamespaceCommand return a namespace sub-command of rootCmd
 func NewNamespaceCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   "namespace [create|append]",
+		Use:   "namespace [create|add|remove]",
```

Comment:
Seem that create is to create the namespace but add/remove is to operate the table namespace. I think it is not proper to put these commands together.